source/_patterns/02-organisms/components/hero-banner.json

```diff
@@ -0,0 +1,33 @@
+{
+  "image": {
```

Comment:
We should add examples of hero banners of other content types. Such as:
- article-poa
- article-vor
- blog-article (inside elife)
- collection
- interview
- labs post
- podcast episode

Reply:
Added 7 hero banner examples for each type